doc/source/data/transforming-data.rst

```diff
@@ -355,13 +356,71 @@ memory your function uses, and prevents Ray from scheduling too many tasks on a
     # Tell Ray that the function uses 1 GiB of memory
     ds.map_batches(uses_lots_of_memory, memory=1 * 1024 * 1024)
 
+
+.. _aggregations:
+
+Aggregations
+============
+
+Ray Data offers out-of-the-box methods for performing aggregations on your data.
+
+These methods include :meth:`~ray.data.Dataset.sum`, :meth:`~ray.data.Dataset.min`, :meth:`~ray.data.Dataset.max`, :meth:`~ray.data.Dataset.mean`, and more
+(see :ref:`API Reference <dataset-api>` for the full list).
+
+To use these methods, call them on your Dataset. For example:
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.range(10)
+    # Schema: {"id": int64}
+    ds.sum(on="id")
+    # 45
+
+
+You can also perform aggregations on grouped data.
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.range(10)
+    # Schema: {"id": int64}
+    ds = ds.add_column("label", lambda x: x % 3)
+    # Schema: {"id": int64, "label": int64}
+    ds.groupby("label").sum(on="id").take_all()
+    # [{'label': 0, 'sum(id)': 18},
+    #  {'label': 1, 'sum(id)': 12},
+    #  {'label': 2, 'sum(id)': 15}]
+
+Each of the above methods also has a corresponding :ref:`AggregateFn <aggregations_api_ref>` object. These objects can be used in
+:meth:`~ray.data.Dataset.aggregate()` or :meth:`Dataset.groupby().aggregate() <ray.data.grouped_data.GroupedData.aggregate>` to compute multiple aggregations at once.
+
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.range(10)
+    # Schema: {"id": int64}
+    ds = ds.add_column("label", lambda x: x % 3)
+    # Schema: {"id": int64, "label": int64}
+    ds.aggregate(ray.data.aggregate.Sum(on="id"), ray.data.aggregate.Count(on="label")).take_all()
```

Comment:
Would be easier to read if put them on separate lines like here

https://github.com/ray-project/ray/blob/d3c76a4327fb6e7f4a23fa179eb66d26809f6069/release/nightly_tests/dataset/tpch_q1.py#L38